src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -43,9 +44,11 @@
       (update :test-preferred-chain-ids chain-ids-string->set)
       (update :type keyword)
       (update :operable keyword)
```

Comment:
Should this be `operable?`

Reply:
No, It's not a boolean. It's a enum `:fully`, `:partially`, and `:no`.

Reply:
Oh okay, [NIT] then maybe `operability` would fit better?

Reply:
I'm not sure, I haven't changed the name of the key for two reasons:
 - It's the existing key name in the account data structure
 - Grammatically, `operable` is correct for the account